### PR TITLE
python3-ruamel.yaml: update to 0.17.21.

### PR DESCRIPTION
**Pull request by TinfoilSubmarine**
#### Testing the changes
- I tested the changes in this PR: **YES**

Tested with all dependents:

- [x] borgmatic
- [x] khard
- [x] python3-ansible-lint
- [x] python3-pykwalify
- [x] whipper (needed patch)

### CODE REVIEW

File: srcpkgs/python3-ruamel.yaml/template [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
+depends="python3-ruamel.yaml.clib"
 short_desc="YAML parser/emitter in Python 3"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="http://yaml.readthedocs.io/"
+changelog="https://sourceforge.net/p/ruamel-yaml/code/ci/default/tree/CHANGES"
 distfiles="${PYPI_SITE}/r/ruamel.yaml/ruamel.yaml-${version}.tar.gz"
-checksum=076cc0bc34f1966d920a49f18b52b6ad559fbe656a0748e3535cf7b3f29ebf9e
+checksum=8b7ce697a2f212752a35c1ac414471dc16c424c9573be4926b56ff3f5d23b7af
+make_check=no
```
Comment (paper42) [inline srcpkgs/python3-ruamel.yaml/template:17]: we should explain why tests are disabled

Comment (TinfoilSubmarine) [reply srcpkgs/python3-ruamel.yaml/template]: Indeed. Fixed

File: srcpkgs/python3-ruamel.yaml/template [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
+depends="python3-ruamel.yaml.clib"
 short_desc="YAML parser/emitter in Python 3"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="http://yaml.readthedocs.io/"
+changelog="https://sourceforge.net/p/ruamel-yaml/code/ci/default/tree/CHANGES"
 distfiles="${PYPI_SITE}/r/ruamel.yaml/ruamel.yaml-${version}.tar.gz"
-checksum=076cc0bc34f1966d920a49f18b52b6ad559fbe656a0748e3535cf7b3f29ebf9e
+checksum=8b7ce697a2f212752a35c1ac414471dc16c424c9573be4926b56ff3f5d23b7af
+make_check=no # no tests included in distfiles
```
Comment (paper42) [inline srcpkgs/python3-ruamel.yaml/template:17]: could we use a distfiles source that has tests?

Comment (TinfoilSubmarine) [reply srcpkgs/python3-ruamel.yaml/template:17]: Not sure if it's possible. It's hosted on sourceforge.net but there are no tarballs/downloads in https://sourceforge.net/projects/ruamel-yaml/files/ like there are for other packages with distfiles from sourceforge, e.g. https://sourceforge.net/projects/drumkv1/files/

**Pull request merged**